app/src/main/java/com/example/kakaowebtoon/presentation/type/TopBarType.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.kakaowebtoon.presentation.type
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import com.example.kakaowebtoon.R
+
+enum class TopBarType(
+    @DrawableRes val firstIconResId: Int,
+    @DrawableRes val secondIconResId: Int,
+    @DrawableRes val thirdIconResId: Int,
+    @StringRes val storageTitleResId: Int? = null,
+    @DrawableRes val mainImageResId: Int? = null,
+    val firstIconStartPadding: Int = 0,
+    val secondIconEndPadding: Int = 0,
+    val thirdIconEndPadding: Int = 0,
```

Comment:
```suggestion
    val firstIconStartPadding: Dp = 0.dp,
    val secondIconEndPadding: Dp = 0.dp,
    val thirdIconEndPadding: Dp = 0.dp,
```
    이렇게하면 나중에 사용할 때 더 편할지도롱룡???